miden-remote-provers/README.md

```diff
@@ -0,0 +1,12 @@
+# Miden remote provers
+
+This crate provides a `RemoteTransactionProver`, a client struct that can be used to interact with the prover service from a Rust codebase.
```

Comment:
I would mention here that the crate distributes the protobuf definitions for the "canonical" remote transaction proving messages, and that `RemoteTransactionProver` is optional based on `tx-prover`.

Reply:
Done.